package/MDAnalysis/analysis/encore/confdistmatrix.py

```diff
@@ -56,7 +56,7 @@ class to compute an RMSD matrix in such a way is also available.
 
 def conformational_distance_matrix(ensemble,
                                    conf_dist_function, selection="",
-                                   superimposition_selection="", n_jobs=1, pairwise_align=True, weights='mass',
+                                   superimposition_selection="", n_jobs=1, max_nbytes='1M', pairwise_align=True, weights='mass',
```

Comment:
document the parameter and include a link to https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html

Reply:
Will this generate the correct documentation?